### ISSUE TITLE
SSD_training: FileNotFoundError: [Errno 2] No such file or directory: '../../frames/frame02374.png'

### ISSUE THREAD

**Issue by jtara1** [rykov8/ssd_keras#86]
In cell 6 of SSD_training.ipynb there's a variable that seems to be a directory.

`path_prefix = '../../frames/'`

And when I run all cells I get the error:

FileNotFoundError: [Errno 2] No such file or directory: '../../frames/frame02374.png'

There's a parameter in the `Generator` class that uses the same value. Changing this value to a real folder doesn't help since the image file isn't found.

Am I missing some data or something?

**Comment by jogundas** [comment]
You have to provide your own data to train on; the author could not include his images due to an NDA, as discussed on some previous Issues.

**Comment by alexiskattan** [comment]
Are there any instructions on how to provide your own data?

**Comment by jogundas** [comment]
@alexiskattan : no, there are no explicit instructions. You might have more luck with pure tensorflow implementation [here](https://github.com/tensorflow/models/tree/master/object_detection).
